.azure-devops/templates/setup-ci-machine.yml

```diff
@@ -1,6 +1,6 @@
 steps:
   - task: PowerShell@2
-    displayName: 'Upgrade pip, install dev_requirements and cryptography'
+    displayName: 'Upgrade pip, install dev_requirements and cryptography, force update azure-mgmt-iothub'
```

Comment:
Just noticed that this is a `displayName` change but doesn't have a corresponding pipeline change - is the pipeline already doing this?

Reply:
oh this was me messing with some pipeline stuff - will remove